styles/style.css

```
@@ -1 +1,311 @@
 /* место для вашего кода */
+.header,
+.footer {
+  display: grid;
+  align-items: center;
+  max-width: 1340px;
+  margin: 92px auto;
+  grid-template: 100% / repeat(3, 1fr);
+}
+/*start of eader*/
+.header__links-list {
+  display: flex;
+  gap: 33px;
+}
+
+.header__links-list-item {
+  display: list-item;
+  font-weight: 400;
+  font-size: 18px;
+  font-style: normal;
+  line-height: normal;
+  list-style: circle;
+}
+
+.header__links-list-item_no-bullit {
+  list-style: none;
+}
+
+.header__link,
+.footer__link {
+  border-bottom: 1px solid;
+  text-decoration: none;
+}
+
+.header__link_active,
+.footer__link_active {
+  border-bottom: none;
+}
+
+.header__logo {
+  text-align: center;
+}
+
+.header__logo-image {
+  width: 148px;
+  height: 105px;
+}
+
+.address {
+  font-size: 18px;
+  font-style: normal;
+  line-height: 1.35;
+}
+.address a {
+  display: inline;
+  border-bottom: 1px solid;
+  line-height: 1;
+  text-decoration: none;
+}
+
+.header__address {
+  justify-self: end;
+}
+/*end of header*/
+
+/*start of section "cover"*/
+.cover {
+  position: relative;
+  max-width: 1440px;
+  margin: 0 auto 180px;
+  background-image: url(../images/cover.jpg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.overlay {
+  position: absolute;
```

При использовании абсолютного позиционирования необходимо указывать сразу две координаты.
Например, `top` и `left`. Если задавать координату только по одной оси - может возникнуть нестандартное отображение в некоторых браузерах, например в браузере Safari.
Причем даже в последних его версиях.

---

Не понимаю замечания. Использую inset: 0; который заменяет указание всех координат одной строчкой.
```suggestion
  .overlay {
  position: absolute;
  inset: 0;
```